src/ert/_c_wrappers/util/substitution_list.py

```diff
@@ -93,8 +93,8 @@ def __getitem__(self, key):
         else:
             raise KeyError(f"No such key:{key}")
 
-    def add_from_string(self, string):
-        self._add_from_string(string)
+    def add_from_string(self, string) -> str:
+        return self._add_from_string(string)
```

Comment:
I think going straight to reimplementing `self._add_from_string` here in python would be a good idea.

Reply:
Semi-straight.